### PR TITLE
Geotools/geo search icon

### PR DESCRIPTION
**Pull request by mdastous-bentley**
Cosmetic changes: New button icon and label.

### CODE REVIEW

File: packages/geo-tools/package.json [hunk 1]
```diff
@@ -46,6 +47,7 @@
     "@bentley/imodeljs-frontend": "^2.9.1",
     "@bentley/imodeljs-i18n": "^2.9.1",
     "@bentley/itwin-client": "^2.9.1",
+    "@bentley/react-scripts": "3.4.6",
```
Comment (aruniverse) [inline packages/geo-tools/package.json:50]: any reason for adding this? i assume you were adding this to pickup the sprite-loader, but looks like that a dependency has also been added

Comment (mdastous-bentley) [reply packages/geo-tools/package.json:50]: "@bentley/react-scripts" is referenced is tsconfig.json.  Without it, its impossible to import SVG files.

Comment (aruniverse) [reply packages/geo-tools/package.json:50]: I believe what you really need from this pkg is the following type declaration. 

```
declare module '*.svg?sprite' {
  const src: string;
  export default src;
}
```

I recommend you create a `geotools/declaration.d.ts` file with the above snippet, and remove the dep on react-scripts. Could you try that and let me know if that works?

**Review by aruniverse — APPROVED** [review]
Approving for now, will revisit later and confirm what the right way to handle svgs are

**Comment by aruniverse** [comment]
@calebmshafer  failing the ado pipelines with the rush change verify step, which i assume is screwed up now from the failing rush publishes earlier due to the pkg.json ordering

**Comment by calebmshafer** [comment]
> @calebmshafer failing the ado pipelines with the rush change verify step, which i assume is screwed up now from the failing rush publishes earlier due to the pkg.json ordering

@aruniverse yes, there is probably PR that needs to be merged with the necessary changes to the changelogs. I'll take a look in a few minutes.

**Review by hnn0003 — APPROVED** [review]

**Pull request merged**